### ISSUE TITLE
Regression: Zeek 6 writes `__bro_plugin__` without trailing newline

### ISSUE THREAD

**Issue by ckreibich** [zeek/zeek#3175]
Prior to Zeek 6 and in current Zeek master the plugin marker file's content ends with a newline. In Zeek 6 it does not. This is hopefully pretty inconsequential but it does trip up some of `zkg`'s tests, which I noticed [here](https://github.com/ckreibich/package-manager/actions/runs/5470041873/jobs/9959648598) while futzing with zeek/package-manager#166.

```
[#2] tests.plugin ... failed
  % 'btest-diff plugins/packages/rot13/__bro_plugin__' failed unexpectedly (exit code 1)
  % cat .diag
  == File ===============================
  Demo::Rot13== Diff ===============================
  --- /dev/fd/63	2023-07-05 23:55:40.732731732 +0000
  +++ /dev/fd/62	2023-07-05 23:55:40.728731693 +0000
  @@ -1 +1 @@
  -Demo::Rot13
  +Demo::Rot13
  \ No newline at end of file
  =======================================
```

Bisecting suggests we introduced the change with 9aeed5284d728baefb5a494c54024b80894ffbca and fixed it in 5f73143e7cf0f98bedabe452583cf23daf309b12.

**Comment by awelzel** [comment]
Looking at the cmake branch for 6.0, when it was pushed, etc, that must have been very unfortunate timing as well.

https://github.com/zeek/cmake/pull/77

**Comment by awelzel** [comment]
This was fixed with 6.0.1.
